packages/mui-base/src/TabsUnstyled/useTabs.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { unstable_useControlled as useControlled } from '@mui/utils';
+
+export interface UseTabsProps {
+  /**
+   * The value of the currently selected `Tab`.
+   * If you don't want any selected `Tab`, you can set this prop to `false`.
+   */
+  value?: string | number | false;
+  /**
+   * The default value. Use when the component is not controlled.
+   */
+  defaultValue?: string | number | false;
+  /**
+   * The component orientation (layout flow direction).
+   * @default 'horizontal'
+   */
+  orientation?: 'horizontal' | 'vertical';
+  /**
+   * The direction of the text.
+   * @default 'ltr'
+   */
+  direction?: 'ltr' | 'rtl';
+  /**
+   * Callback invoked when new value is being set.
+   */
+  onChange?: (event: React.SyntheticEvent, value: number | string) => void;
+  /**
+   * If `true` the selected tab changes on focus. Otherwise it only
+   * changes on activation.
+   */
+  selectionFollowsFocus?: boolean;
+}
+
+function useUniquePrefix() {
```

Comment:
Can we just use `useId` instead?

Reply:
Yep, changed, I copied this from the lab directly